client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/hooks/useOutboundLineEditRows.ts

```diff
@@ -31,6 +35,7 @@ export const useOutboundLineEditRows = (
     const onHoldRows: DraftOutboundLine[] = [];
     const noStockRows: DraftOutboundLine[] = [];
     const wrongPackSizeRows: DraftOutboundLine[] = [];
+    const incorrectBatchRows: DraftOutboundLine[] = [];
```

Comment:
should this be `notScannedRows` ?

Reply:
I don't think `notScannedRows` gives enough detail. If we're changing then I've gone with `batchAndScannedBatchMismatchRows` 🤷

Reply:
Hmm, not sure if `batchAndScannedBatchMismatchRows` is better (i can't quite figure out what it means by reading it), from reading the code it means "rows to be disabled because they don't match scanned batch", `scannedBatchMismatchRows` ?